### PR TITLE
Remove UJS

### PR DESCRIPTION
**Pull request by dem4ron**
Screenshot 2023-09-06 at 11 33 56

https://github.com/exercism/website/assets/66035744/69b4e092-4398-4017-9500-f49629e4939b

### CODE REVIEW

File: app/helpers/view_components/user_menu.rb [hunk 1]
```diff
@@ -38,7 +38,7 @@ def menu_items
         { html: link_to("Your Journey", Exercism::Routes.journey_path), className: "opt" },
         { html: link_to("Settings", Exercism::Routes.settings_path), className: "opt" },
         ({ html: link_to("Maintaining", Exercism::Routes.maintaining_root_path), className: "opt" } if current_user.maintainer?),
-        { html: link_to("Sign out", Exercism::Routes.destroy_user_session_path, method: :delete), className: "opt" }
+        { html: button_to("Sign out", Exercism::Routes.destroy_user_session_path, method: :delete), className: "opt" }
```
Comment (iHiD) [inline app/helpers/view_components/user_menu.rb:41]:
```
git grep link_to | grep method

app/views/settings/show.html.haml:          = link_to user_confirmation_path(user: { email: current_user.unconfirmed_email }), method: :post, class: "c-prominent-link" do
```

File: app/javascript/packs/core.tsx [hunk 2]
```diff
@@ -1,10 +1,4 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-require('@rails/ujs').start()
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-require('@rails/activestorage').start()
```
Comment (iHiD) [inline app/javascript/packs/core.tsx:4]: Can we remove these from package.json too?

Comment (ErikSchierboom) [reply app/javascript/packs/core.tsx:4]: And remove the comments?

Comment (dem4ron) [reply app/javascript/packs/core.tsx:4]: yes

**Pull request merged**